centos8-test-container/Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM centos:centos8
+
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+rm -f /lib/systemd/system/multi-user.target.wants/*; \
+rm -f /etc/systemd/system/*.wants/*; \
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*; \
+rm -f /lib/systemd/system/anaconda.target.wants/*;
+
+RUN yum clean all && \
+    yum -y install epel-release && \
+    yum -y install \
+    # python3-httplib2 \
+    # python3-keyczar \
+    # python3-mock \
+    # python3-passlib \
```

Comment:
I'd check to see if these are still needed, and if so, by which tests. Integration tests should install these if they're not already installed.

Reply:
I did find that `setup_passlib` is a role that is being used. I didn't find any evidence of other `setup` roles. I'm not sure of the best way to tell if tests need this or not besides running and see what breaks.

I'll just remove these from the base image.